docs/_components/icon-list.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Icon List
+lead: An icon list reinforces the meaning and visibility of individual list items with a leading icon.
+---
+
+{% include helpers/base-component.html component="icon-list" %}
+
+## Default
+
+{% capture example %}
+    <ul class="usa-icon-list usa-icon-list--size-md">
```

Comment:
Including `usa-icon-list` and `usa-icon-list--size-md` was enough to fix the padding issue. Thanks for the help on this!